config.toml

```diff
@@ -1,10 +1,7 @@
-[API]
-    Timeout = "5m"
-
 [Storage]
     [Storage.File]
         [Storage.File.CSV]
             Format = "CSV"
             Path = "/tmp/data"
-            OmitHeader = true
-            FilePattern = "{jobname}-{table}.csv"
+            OmitHeader = false
```

Comment:
We can always omit it and it makes simpler to load and match the data if anything changes.

Reply:
How are you planning on removing the header?

Reply:
We can omit headers when loading things in BigQuery. Same with other things like DuckDB/Arrow/Pandas.

Reply:
Yeah I know 😅 I'm just wondering what's the value add of keeping the headers rather than omitting it, since with those tools you have to specify omitting the headers at every load whereas this is already configured once so it's less overhead.

Reply:
I guess the value is that headers are explicit rather than defined in other places. We don't need to know the schema or order of columns to explore or load the files. We can `select * from gs://...`. 

Lets say the schema changes from `height, cid, params` to `height, cid, other_thing`. Having the explicit headers should make that very easy to detect. It won't probably happen but a few rows every run give us more information about the data.